TOC-tidb-cloud.md

```diff
@@ -123,6 +123,7 @@
   - [Overview](/tidb-cloud/tidb-cloud-tune-performance-overview.md)
   - Analyze Performance
     - [Statement Analysis](/tidb-cloud/tune-performance.md)
+    - [Slow Queries](/tidb-cloud/tune-performance.md#slow-queries)
     - [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer)
     - [Statement Summary Tables](/statement-summary-tables.md)
```

Comment:
```suggestion
    - [Use the Diagnosis Tab](/tidb-cloud/tune-performance.md)
    - [Use Statement Summary Tables](/statement-summary-tables.md)
```

Reply:
Currently, the docs website does not support using subsections in the TOC file any more. There is a CI error reported.